provider/azure/azure_image.go

```diff
@@ -65,27 +64,6 @@ func (a *Azure) getArchiveName(ctx *lepton.Context) string {
 // CustomizeImage returns image path with adaptations needed by cloud provider
 func (a *Azure) CustomizeImage(ctx *lepton.Context) (string, error) {
 	imagePath := ctx.Config().RunConfig.ImageName
-	symlink := filepath.Join(filepath.Dir(imagePath), "disk.raw")
-
-	if _, err := os.Lstat(symlink); err == nil {
-		if err := os.Remove(symlink); err != nil {
-			return "", fmt.Errorf("failed to unlink: %+v", err)
-		}
-	}
-
-	err := os.Link(imagePath, symlink)
-	if err != nil {
-		return "", err
-	}
-
-	archPath := filepath.Join(filepath.Dir(imagePath), a.getArchiveName(ctx))
-	files := []string{symlink}
-
-	err = lepton.CreateArchive(archPath, files)
```

Comment:
was all of this accidentally removed?

Reply:
I don't see the archive used anywhere in azure. That function returns the original raw image path.